File: app/src/main/java/prism/mediscan/model/Interaction.java
package prism.mediscan.model;

/**
 * Created by joshua-hugo on 11/02/18.
 */

public class Interaction {
    private String description;
    private String type;
    private String conseil;

    public Interaction(String description, String type, String conseil){
        this.description = description;
        this.type = type;
        this.conseil = conseil;
    }

    public String getDescription(){
        return description;
    }

    public String getType(){
        return type;
    }

    public String getConseil(){
        return conseil;
    }
}

File: app/src/main/java/prism/mediscan/model/Interaction.kt
package prism.mediscan.model

import java.io.Serializable

/**
 * Created by rapha on 11/02/2018.
 */
data class Interaction(val description: String, val type: String, val conseil: String, val cis1: String, val cis2: String, val nomMedicament1: String, val nomMedicament2: String) : Serializable
